src/css-loader-core/parser.js

```diff
@@ -62,19 +70,74 @@ export default class Parser {
     exportNode.remove();
   }
 
-  fetchImport(importNode, relativeTo, depNr) {
-    let file = importNode.selector.match(importRegexp)[1],
-      depTrace = this.trace + String.fromCharCode(depNr);
-    return this.pathFetcher(file, relativeTo, depTrace).then(
-      (exports) => {
-        importNode.each((decl) => {
-          if (decl.type == "decl") {
-            this.translations[decl.prop] = exports[decl.value];
-          }
-        });
-        importNode.remove();
-      },
-      (err) => console.log(err)
-    );
+  async fetchImport(importNode, relativeTo, depNr) {
+    const file = importNode.selector.match(importRegexp)[1].replace(/^["']|["']$/g, ""),
```

Comment:
Let's split the declarations into separate statements everywhere for the readability sake